Description: Add UPS_DRIVER environment variable during udev configuration.
 .
 nut (2.6.5-1.netgear1) unstable; urgency=low
 .
   * Non-maintainer upload.
   * Add UPS_DRIVER environment variable during udev configuration.
Author: Justin Maggard <justin.maggard@netgear.com>

---

diff --git a/tools/nut-usbinfo.pl.orig b/tools/nut-usbinfo.pl
index 989cef2..5d91ca1 100755
--- a/tools/nut-usbinfo.pl.orig
+++ b/tools/nut-usbinfo.pl
@@ -159,7 +159,7 @@ sub gen_usb_files
 			print $outUdev "# ".$vendor{$vendorId}{$productId}{"comment"}.' - '.$vendor{$vendorId}{$productId}{"driver"}."\n";
 			print $outUdev "ATTR{idVendor}==\"".removeHexPrefix($vendorId);
 			print $outUdev "\", ATTR{idProduct}==\"".removeHexPrefix($productId)."\",";
-			print $outUdev ' MODE="664", GROUP="@RUN_AS_GROUP@"'."\n";
+			print $outUdev ' MODE="664", GROUP="@RUN_AS_GROUP@"'.", ENV{UPS_DRIVER}=\"$vendor{$vendorId}{$productId}{'driver'}\""."\n";
 			
 			# devd device entry
 			print $out_devd "# ".$vendor{$vendorId}{$productId}{"comment"}.' - '.$vendor{$vendorId}{$productId}{"driver"}."\n";
